client/core/trade.go

```diff
@@ -120,7 +120,8 @@ type matchTracker struct {
 	// send an initialization until it is confirmed with the server (see
 	// authDEX) that the match is not revoked. This should be set on reconnect
 	// for all taker matches in MakerSwapCast. Guarded by exceptionMtx.
-	checkServerRevoke bool
+	checkServerRevoke                                              bool
+	needsDynamicSwapFeeConfirms, needsDynamicRedemptionFeeConfirms bool
```

Comment:
I think we can get rid of these fields. In `(*Core).tick`, they can be local variables. And as @chappjc pointed out, our `MatchConfirmed` flag should preclude any `MatchIsActive` checks.

Reply:
trying...

Reply:
All I can think of is adding a flag to the db for whether this match's fee's have been accounted for, which is probably the best answer so will do that. Hope I'm not missing something obvious here.

Reply:
Removed them https://github.com/decred/dcrdex/pull/1752/commits/66115041bde7a69977cbb435870f8889077d82a9

Reply:
I'm not seeing the need for the new MatchProof fields stored in the DB.
I thought it was fine to simply require the redemption conf requirement to be no less than the fee update conf requirement.

Reply:
Oh if we restarted, or didn't have a flag attached to the matchTracker somewhere, we'd repeatedly recompute and add the fees.  Dang.
New DB data it is then.  That's unfortunate.